rpc-server/src/modules/receipts/methods.rs

```diff
@@ -36,6 +36,38 @@ pub async fn receipt(
     Ok(result.map_err(near_jsonrpc::primitives::errors::RpcError::from)?)
 }
 
+/// Fetches a receipt record by it's ID
+#[cfg_attr(feature = "tracing-instrumentation", tracing::instrument(skip(data)))]
+pub async fn view_receipt_record(
+    data: Data<ServerContext>,
+    Params(params): Params<serde_json::Value>,
+) -> Result<readnode_primitives::ReceiptRecord, RPCError> {
+    tracing::debug!("`view_receipt_record` call. Params: {:?}", params);
+    let receipt_request =
+        near_jsonrpc::primitives::types::receipts::RpcReceiptRequest::parse(params)?;
+    crate::metrics::RECEIPT_RECORD_REQUESTS_TOTAL.inc();
+
+    #[cfg(feature = "shadow_data_consistency")]
+    {
+        let shadow_result = crate::utils::shadow_compare_results_handler(
+            crate::metrics::RECEIPT_RECORD_REQUESTS_TOTAL.get(),
+            data.shadow_data_consistency_rate,
+            &fetch_receipt_record(&data, &receipt_request).await,
+            data.near_rpc_client.clone(),
```

Comment:
The entire shadow data consistency check is redundant here. There is no such method on native RPC, so there is nothing to compare. Please drop it.